base/strings/io.jl

```diff
@@ -10,7 +10,7 @@ if `io` is not given) a canonical (un-decorated) text representation.
 The representation used by `print` includes minimal formatting and tries to
 avoid Julia-specific details.
 
-`print` falls back to calling `show`, so most types should just define
+`print` falls back to calling the 2-argument `show(io, x)` for each argument `x` in `xs`, so most types should just define
```

Comment:
I think this paragraph should be reflowed with the added content, but otherwise looks good.